src/components/QuorumPicker.jsx

```diff
@@ -23,167 +23,149 @@ import "./styles.css";
 
 const MAX_TOTAL_SIGNERS = 7;
 
-class QuorumPicker extends React.Component {
-  renderIncrementRequiredSigners = () => {
-    const { requiredSigners, totalSigners, frozen } = this.props;
+const QuorumPicker = ({
+  frozen,
+  requiredSigners,
+  setRequiredSigners,
+  totalSigners,
+  setTotalSigners,
+}) => {
+  const handleIncrementRequiredSigners = (event) => {
+    setRequiredSigners((prev) => prev + 1);
+    event.preventDefault();
+  };
+
+  const handleDecrementRequiredSigners = (event) => {
+    setRequiredSigners((prev) => prev - 1);
+    event.preventDefault();
+  };
```

Comment:
This should be reverted to the way it was previously. You don't know for sure that the `setRequiredSigners` function is going to be a `setState` function, since it's passed as a prop, so you shouldn't use the `prev` syntax. The same goes for each of these handle increment/decrement functions.
```suggestion
  const handleDecrementRequiredSigners = (event) => {
    setRequiredSigners(requiredSigners - 1);
    event.preventDefault();
  };
```